### PR TITLE
RHIDP-3016 Fix Enabling authentication section in Admin guide

### PR DESCRIPTION
**Pull request by themr0c**
* Disabled unsupported authentication module: Gitlab.
* Moved in the section supported authentication modules: GitHub, OIDC.

**Version(s):**
1.2

**Issue:**

https://issues.redhat.com/browse/RHIDP-3016

**Link to docs preview:**

**Reviews:**
- [ ] SME: @jasperchui 
- [x] QE: @albarbaro 
- [x] Docs review: @linfraze / @hmanwani-rh / @Gerry-Forde 

**Additional information:**

*

### CODE REVIEW

**Review by linfraze — APPROVED** [review]

File: assemblies/assembly-enabling-authentication.adoc [hunk 1]
```diff
@@ -35,4 +35,13 @@ For each provider that you want to use, follow the dedicated procedure to comple
...
-include::modules/auth/proc-adding-gitlab-oauth-as-an-authentication-provider.adoc[leveloffset=+1]
+
+
+include::assembly-auth-provider-github.adoc[leveloffset=+1]
+
+
+include::assembly-auth-provider-oidc.adoc[leveloffset=+1]
+
+
+//include::modules/auth/proc-adding-gitlab-oauth-as-an-authentication-provider.adoc[leveloffset=+1]
```
Comment (hmanwani-rh) [inline assemblies/assembly-enabling-authentication.adoc:46]: Can we also add the GitHub and OIDC auth providers sections from the Getting Started guide to this assembly?

**Review by hmanwani-rh — COMMENTED** [review]
@themr0c Can we include the existing GitHub and OIDC sections from the Getting Started guide in the 'assembly-enabling-authentication.adoc'?

**Comment by briandooley** [comment]
RHDO docs Triage: [This comment](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/366#pullrequestreview-2185110855) will be addressed in restructuring.

File: assemblies/assembly-enabling-authentication.adoc [hunk 2]
```diff
@@ -35,4 +35,13 @@ For each provider that you want to use, follow the dedicated procedure to comple
 . Configure {product} to use the authentication provider.
 
 include::proc-adding-azure-as-an-authentication-provider.adoc[leveloffset=+1]
-include::modules/auth/proc-adding-gitlab-oauth-as-an-authentication-provider.adoc[leveloffset=+1]
```
Comment (kadel) [inline assemblies/assembly-enabling-authentication.adoc:38]: Why is this being removed?

**Review by kadel — APPROVED** [review]

File: assemblies/assembly-enabling-authentication.adoc (hunk 1 above)
Comment (linfraze) [reply assemblies/assembly-enabling-authentication.adoc:46]: The associated Jira (RHIDP-3016) calls for the following actions: 

- Replace the Enabling the GitLab OAuth authentication module with Enabling the GitHub OAuth authentication provider
- Add an Enabling Keycloak with OIDC authentication module to the assembly (currently in the Getting Started guide) 

It looks like this change resolves the issue.

**Review by linfraze — APPROVED** [review]
/lgtm

**Pull request merged**